### PR TITLE
Support for importing mtga decks.

### PR DESCRIPTION
**Pull request by hitch17**

### CODE REVIEW

File: Mage/src/main/java/mage/cards/decks/importer/MtgaImporter.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class MtgaImporter extends PlainTextDeckImporter {
+
+    private static final Map<String, String> SET_REMAPPING = ImmutableMap.of("DAR", "DOM");
+    private static final Pattern MTGA_PATTERN = Pattern.compile(
+            "(\\p{Digit}+)" +
+            "\\p{javaWhitespace}+" +
+            "([ !\"&',\\-./0-9:A-Za-zé]+)" +
```
Comment (JayDi85) [inline Mage/src/main/java/mage/cards/decks/importer/MtgaImporter.java]: Why it's have only `é`? There are many other skip symbols (xmage don't use it at all, but card names have). Replace list from inner tool:
```
      Replace(s, 'â', 'a');
      Replace(s, 'á', 'a');
      Replace(s, 'í', 'i');
      Replace(s, 'û', 'u');
      Replace(s, 'Ã†', 'Ae');
      Replace(s, 'Ã¶', 'o');
      Replace(s, 'é', 'e');
      Replace(s, 'à', 'a');
      Replace(s, 'ú', 'u');
```

Comment (JayDi85) [reply Mage/src/main/java/mage/cards/decks/importer/MtgaImporter.java]: P.S. `TxtDeckImporter` have replace list too

Comment (hitch17) [reply Mage/src/main/java/mage/cards/decks/importer/MtgaImporter.java]: I think it needs to allow 'é' for `Senator Padmé Amidala` in StarWars.java:293.  I'm not sure if that should actually be changed from 'é' to 'e' though.

Comment (JayDi85) [reply Mage/src/main/java/mage/cards/decks/importer/MtgaImporter.java]: Yes, Senator Padmé was wrong (I'v fixed that in last commit). Xmage's names must contain only simple ascii symbols cause it's uses for images file names in different os.

Comment (hitch17) [reply Mage/src/main/java/mage/cards/decks/importer/MtgaImporter.java]: That makes sense. I'll remove 'é' and add the replacements.

**Pull request merged**